packages/ui/src/components/ActionBar/style.less

```diff
@@ -8,6 +8,10 @@
 @condo-action-bar-width: calc(100% + @condo-action-bar-overflow-width * 2);
 @condo-action-bar-large-screen-bottom-indent: @condo-global-spacing-24;
 
+.condo-affix-placeholder {
+  height: @condo-action-bar-large-screen-bottom-indent;
```

Comment:
what about small screens? will this work well?

Reply:
good comment, let me double check it